cli/src/processSourceFiles.ts

```diff
@@ -69,13 +69,18 @@ const injectBundleIDToSourceFile = (sourceFile: string, bundleID: string) => {
   return jsLines.join('\n');
 };
 
-const findJSFilesRecursively = (
+const extractSourceMapUrl = (jsContent: string): string | null => {
+  const match = jsContent.match(/\/\/[@#] sourceMappingURL=(.+)$/m);
```

Comment:
I would probably use:
```suggestion
  const match = jsContent.match(/^\/\/[@#] sourceMappingURL=(.+)$/m);
```

which I believe should perform better for long lines that don't have this text